### ISSUE TITLE
Using the logged on user's profile for Chrome

### ISSUE THREAD

**Issue by sei-dupdyke** [cmu-sei/GHOSTS#122]
### Discussed in https://github.com/cmu-sei/GHOSTS/discussions/121

<div type='discussions-op-text'>

<sup>Originally posted by **Cyb3r-Monk** September  6, 2022</sup>
Chrome uses a temporary profile like below:
`C:\Users\S50FB~1.CHI\AppData\Local\Temp\scoped_dir18316_422758565\Default`

Is it possible to specify the logged on user's profile so that history and other data remains? </div>

**Comment by sei-dupdyke** [comment]
This is up there with version 6.1.26 (which is available as current).

This feature needs a timeline entry under handlerArgs called "user-data-dir":

```
 "HandlerType": "BrowserChrome",
      "HandlerArgs": {
          "isheadless": "false",
          "incognito": "false",
          "user-data-dir": "C:\\Users\\david.lightman\\special_profile_location",
          ...
```

Incognito probably needs to be set to false, but I did not test it.

This flag does not currently work in FireFox, cursory searches seem to indicate there is some problem with geckodriver related to specifying profile paths (aka "I could not get this to work in a reasonable timeframe at the end of a long day"). I could revisit if there is call for it...

**Comment by Cyb3r-Monk** [comment]
I tried using `%LOCALAPPDATA%` but it gave the below error. I don't know if it's possible to specify the path like that (would be much more convenient).   

<img width="393" alt="image" src="https://user-images.githubusercontent.com/26627224/188960493-b46e4ca9-940b-4d75-b86f-b2c2d5f8b860.png">

Below config works properly (don't put `Default` at the end):
`"C:\\Users\\user.name\\AppData\\Local\\Google\\Chrome\\User Data\\"`

**Comment by sei-dupdyke** [comment]
Environment variables are resolved in config paths for version 6.1.27.
